force-app/main/default/classes/CON_DoNotContact_TDTM.cls

```diff
@@ -58,8 +58,8 @@ public class CON_DoNotContact_TDTM extends TDTM_Runnable {
         for (integer i=0; i<newlist.size();i++) {
             Contact con = (Contact)newlist[i];
             if (triggerAction == TDTM_Runnable.Action.BeforeInsert) {
+                AccountAdapter.enableHouseholdDeceasedUpdate(true);
```

Comment:
Just for documentation purposes, we needed to move this line outside of the `if` block because the deceased flag was not updating when a new living household member was inserted which would cause the household to no longer be marked deceased.